client/my-sites/sites/index.jsx

```diff
@@ -37,7 +37,7 @@ class Sites extends Component {
 		}
 
 		// No support for Gutenberg on VIP.
-		if ( /^\/block-editor/.test( path ) ) {
+		if ( /^(\/post|\/page|\/edit)/.test( path ) ) {
```

Comment:
I think we don't want to exclude VIP sites here anymore. This component renders a site selector when accessing routes without a site param:

<img width="400" alt="Screen Shot 2020-10-14 at 11 10 42" src="https://user-images.githubusercontent.com/1233880/95968597-eb4d2f80-0e0d-11eb-9def-20d001801caa.png">

We were excluding VIP sites on `/block-editor` routes because there was no support for Gutenberg for them, but site selectors rendered on `/post` or `/page` routes should include VIP sites because that's a working route for them (I guess end up using the classic editor).

Reply:
makes sense